packages/wallet-sdk/src/relay/WalletLinkRelay.ts

```diff
@@ -101,6 +99,9 @@ export class WalletLinkRelay extends WalletSDKRelayAbstract {
 
   constructor(options: Readonly<WalletLinkRelayOptions>) {
     super();
+    this.resetAndReload = this.resetAndReload.bind(this);
+    this.handleIncomingEvent = this.handleIncomingEvent.bind(this);
+
```

Comment:
following the existing pattern
![image](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/a0353d06-7602-414d-9550-1b758150ac4d)